bot_python_sdk/store.py

```diff
@@ -1,76 +1,107 @@
 import json
 import os
+from bot_python_sdk.logger import Logger
 
 _actions_file_path = 'storage/actions.json'
 _last_triggered_file_path = 'storage/last_triggered.json'
 _qr_image_path = 'storage/qr.png'
 _configuration_file_path = 'storage/configuration.json'
 _bot_public_key = 'storage/public.pem'
 
+LOCATION = 'Store'
+
 
 class Store:
+    def __init__(self):
```

Comment:
is there a reason/benefit to declaring the empty constructor here?

Reply:
The empty constructor helps to know that the class is being initialized in the other py files, and making the initialization context more clear. This helps us to know if any variable is initialized or not. In future if we want to add any variables then we can .

Reply:
I don't agree, if we need to add variables in the future we can add the constructor then. Now it is just not adding any value imho. So please remove it.

Reply:
i have removed the empty constructor.